clang/include/clang/ASTMatchers/ASTMatchers.h

```diff
@@ -4199,7 +4199,7 @@ AST_MATCHER_P_OVERLOAD(QualType, references, internal::Matcher<Decl>,
 ///     cxxRecordDecl(hasName("Y")))))
 ///   matches `y.m()`, `x.m()` and (g()).m(), but not `x.g()`.
```

Comment:
Since we're cleaning stuff up in this area: 
```suggestion
///   matches `y.m()`, `x.m()` and (`g()).m()`, but not `x.g()`).
```